components/post_view/message_attachments/message_attachment/index.js

```diff
@@ -5,9 +5,19 @@ import {connect} from 'react-redux';
 import {bindActionCreators} from 'redux';
 
 import {doPostActionWithCookie} from 'mattermost-redux/actions/posts';
+import {getCurrentRelativeTeamUrl} from 'mattermost-redux/selectors/entities/teams';
+
+import {handleFormattedTextClick} from "utils/utils";
 
 import MessageAttachment from './message_attachment';
 
+function mapStateToProps(state) {
+    return {
+        handleFormattedTextClick: (e) =>
```

Comment:
You'll also want to pass the relative team URL into here instead of passing in a new function